Mange team/domain matching docs

ngrayluna opened the pull request:
## Description

Refactored Access management section:
1. Consolidated and update content in [Manage users](https://docs.wandb.ai/guides/hosting/iam/manage-users) and added said content to a new page called [Managed your organization](https://bedb66be.docodile.pages.dev/guides/hosting/iam/manage-organization).
2. Added a landing page for [Access management](https://bedb66be.docodile.pages.dev/guides/hosting/iam/access-management-intro).

Pages to review: 
* [Manage your organization](https://bedb66be.docodile.pages.dev/guides/hosting/iam/manage-organization)
   * Domain matching content can be found here: https://bedb66be.docodile.pages.dev/guides/hosting/iam/manage-organization#automatically-add-new-users-to-a-team
* Access management landing page [Do not review yet]

To do: 
* Add intro blurb on Access management page. 
* Possibly add more images to Manage your organization to break up the wall of text.
* Follow up PR for these subsections (See commented out content at the bottom of `manage-organization.md` file):
   * Manage team storage
   * Create and assign custom roles
   * Privacy  

## Future URL

Assuming we don't change the page name, the future URL will be:

`https://docs.wandb.ai/guides/hosting/iam/manage-organization`

## Ticket

https://wandb.atlassian.net/browse/DOCS-962

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [ ] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [ ] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [ ] I merged the latest changes from `main` into my feature branch before submitting this PR.

ngrayluna committed:
Add Manage your org doc and landing page for extra content...WIP

ngrayluna committed:
Added Change org name section

github-actions[bot] committed:
Optimised images with calibre/image-actions

ngrayluna committed:
Filled intro para, updated image

ngrayluna committed:
resolved merge conflicts

github-actions[bot] committed:
Optimised images with calibre/image-actions

ngrayluna committed:
Added SSO info

ngrayluna committed:
Merge branch 'domain_match_docs' of https://github.com/wandb/docodile into domain_match_docs

ngrayluna committed:
v0 landing page

cassface commented on docs/guides/hosting/iam/manage-organization.md:
```diff
@@ -0,0 +1,227 @@
+
+## Change the name of your organization
+
+1. Navigate to the organization's dashboard at https://wandb.ai/home.
```
I believe this is incorrect, for both SaaS and server... 

There's no org display name in server today.

In SaaS the org display name is editable via https://wandb.ai/account-settings/wandb/settings for customers this pattern would be https://wandb.ai/account-settings/{Org Name}/settings

ngrayluna replied:
@cassface What's the official name for `https://wandb.ai/home` ?  

In the instructions, this is step 1 but then in the subsequent steps I describe to the go to the Settings page. etc

cassface commented on docs/guides/hosting/iam/manage-organization.md:
```diff
@@ -0,0 +1,227 @@
+
+## Add and manage users
+
+Use your organization's dashboard to invite users, assign or update a user's role, remove users from your organization, assign the billing administrator, and more. Your organization's dashboard is located at: https://wandb.ai/home.
```
our organization's dashboard is located at: https://wandb.ai/home.

This is incorrect. see above comment

cassface commented on docs/guides/hosting/iam/manage-organization.md:
```diff
@@ -0,0 +1,227 @@
+
+### Invite a user
+
+1. Navigate to your organization's dashboard at https://wandb.ai/home.
```
This needs to be corrected

ngrayluna committed:
resolved merge conflicts

ngrayluna committed:
Removed page again..

ngrayluna committed:
quick edits

ngrayluna commented on docs/guides/hosting/iam/manage-organization.md:
```diff
@@ -0,0 +1,229 @@
+Reach out to your W&B for further assistance.
+:::
+
+### Auto provision users
```
@cassface Is auto provisioning applicable for Multi tenant? Or only Dedicated and/or Self managed?

cassface commented on docs/guides/hosting/iam/manage-organization.md:
```diff
@@ -0,0 +1,229 @@
+displayed_sidebar: default
+---
+
+# Mange your organization
```
``` suggestion
# Manage your organization
```

ngrayluna committed:
Updates from Notion doc v1

cassface committed:
Update manage-organization.md

cassface committed:
Update manage-organization.md

1. clarified that organization names could only be changed in SaaS cloud

cassface committed:
Update manage-organization.md

Move domain capture under Organization management as it's one of three ways customers can add users to their organization

cassface committed:
Update manage-organization.md

small grammar fix. 

removed sentence that implied all users would become members of the default team

cassface committed:
Update manage-organization.md

added that auto provisioning is only for Dedicated cloud instances and Self-managed deployments as a call out

ngrayluna committed:
Moved domain capture

ngrayluna committed:
Merge branch 'main' into domain_match_docs

ngrayluna committed:
fix broken links

ngrayluna committed:
Merge branch 'domain_match_docs' of https://github.com/wandb/docodile into domain_match_docs

ngrayluna committed:
Added abhinavs feedback

cassface committed:
Update manage-organization.md

added Domain claiming documentation for single tenant instances. 

added invited user example to domain claiming.

Because teams by default have domain claiming enabled, I reordered the steps so that enabling the feature was listed first, and customizing team settings was second

cassface committed:
Update manage-organization.md

moved requirement that domains must be unique to the right place

ngrayluna committed:
Fixed notes

ngrayluna committed:
Resolved merge conflict

ngrayluna committed:
Added steps for Dedicated to invite user

cassface committed:
Update manage-organization.md

added a single sentence intro for "invite a user"

used tabs to separate domain claiming docs for SaaS vs single-tenant

ngrayluna committed:
Added sentence to invite a user

ngrayluna committed:
Merge branch 'domain_match_docs' of https://github.com/wandb/docodile into domain_match_docs

ngrayluna committed:
Moved info blurb, clarified dedicated blurb

cassface committed:
Update manage-organization.md

added table for domain capture behavior on dedicated

ngrayluna committed:
Merge branch 'main' into domain_match_docs

ngrayluna committed:
Merge branch 'domain_match_docs' of https://github.com/wandb/docodile into domain_match_docs

ngrayluna committed:
Fix table

ngrayluna committed:
Removed via

ngrayluna committed:
Fixed dashboard URL

cassface committed:
Update manage-organization.md

updated the SSO section to make it clear it was available on SaaS too
